### PR TITLE
Convert all path sep to unix in config. Fixes #121

### LINKED ISSUE #121: Windows paths stored in webgme-setup.json fails on unix.
When depending on a component from a "windows-repo" the adding fails with error due to the windows path separator.

### PR DESCRIPTION
**Pull request by brollb**

### CODE REVIEW

**Comment by brollb** [comment]
@pmeijer - Do you mind checking this out?

File: src/DecoratorManager.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ DecoratorManager.prototype.new = function(options, callback) {
 
         // Get the src, test paths
         var setupConfig = R.mapObjIndexed(function(empty, type) {
-            return path.join(type, 'decorators', name+'Decorator');
+            return `${type}/decorators/${name}Decorator`;
```
Comment (pmeijer) [inline src/DecoratorManager.js]: I've never seen this before (the `'${var}'`..).
Is it some built in template?

Comment (brollb) [inline src/DecoratorManager.js]: I am using ES6 and using babel to cross compile down to es5. These are ES6 string templates - basically they are strings with interpolation. ie, `'hello ' + name + ', my name is ' + me` is the same as `'hello ${name}, my name is ${me}'`

Comment (pmeijer) [inline src/DecoratorManager.js]: Cool, had a feeling that was the case. Thanks for the fix btw!

**Pull request merged**